src/styles/main.scss

```diff
@@ -1,3 +1,71 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__wrap {
+    display: flex;
+
+    flex-wrap: wrap;
+    max-width: $day-width * $day-count + $calendar-padding;
+    padding: $calendar-padding;
+    gap: 1px;
+
+    &--month-length{
+      @for $days-in-month from 28 through 31 {
+        &--#{$days-in-month} > :nth-last-child(-n + #{31 - $days-in-month}) {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__day {
+    background-color: $main-color;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: $day-color-hover;
+      cursor: pointer;
+
+      transform: translateY(-20px);
+    }
+
+    @for $num from 1 through 31 {
+      &:nth-child(#{$num})::before {
+        content: "#{$num}";
+        color: #000;
+
+        border: 1px solid black;
+        box-sizing: border-box;
+
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        width: $day-width;
+        height: $day-height;
+      }
+    }
+
+    $week: mon, tue, wed, thu, fri, sat, sun;
+
+    @each $day in $week {
+      &--start-day--#{$day} {
```

Comment:
should be related to calendar (in the similar way you did for `month-length modificator)